pkg/operator/ceph/csi/controller.go

```diff
@@ -194,6 +194,13 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 		return opcontroller.ImmediateRetryResult, errors.Wrap(err, "failed to create pool ID mapping config map")
 	}
 
+	exists, err := checkCsiCephConfigMapExists(r.opManagerContext, r.context.Clientset, r.opConfig.OperatorNamespace)
```

Comment:
How about if we create the configmap in at least one of the integration test suites? Then we can get some test coverage that the example works end to end both with the custom overrides and without.

Reply:
@travisn where exactly I can create this configmap in the integration test suite?

Reply:
How about in [CreateCephCluster()](https://github.com/rook/rook/blob/9840273d8e82ab5077f298aa7a6af11a801c4299/tests/framework/installer/ceph_installer.go#L238) next to where the configmap is created for the other ceph.conf overrides.

Reply:
Done. Thanks for the hint